src/renderer/services/chain/fees/deposit.ts

```diff
@@ -92,11 +108,31 @@ const depositFeeByChain$ = ({
     }
     case ETHChain: {
       return FP.pipe(
-        sequenceSOption({ recipient, amount, memo }),
+        sequenceSOption({ recipient, router, amount, memo }),
         O.fold(
           () => Rx.of(RD.initial),
-          ({ recipient, amount, memo }) =>
-            ETH.fees$({ recipient, amount, memo }).pipe(liveData.map((fees) => fees.fast))
+          ({ recipient, router, amount, memo }) =>
+            ETH.callFees$(
+              router,
+              ethRouterABI,
+              'deposit',
+              isEthAsset(asset)
+                ? [
+                    FP.pipe(getEthChecksumAddress(recipient), O.toUndefined),
+                    ETHAddress,
+                    0,
+                    memo,
+                    {
+                      value: amount.amount().toFixed()
+                    }
+                  ]
+                : [
+                    ethers.utils.getAddress(recipient.toLowerCase()),
+                    FP.pipe(getEthTokenAddress(asset), O.toUndefined),
+                    amount.amount().toFixed(),
+                    memo
```

Comment:
Why two different functions to get addresses for receipient: `getEthChecksumAddress` + `ethers.utils.getAddress`?